### PR TITLE
dashboard: support set config to disable token

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve?

dashboard support set config to disable token

### What is changed and how does it work?

add a config for disabling token, if it is true, will use the local client.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/clientpool/client.go [hunk 1]
```diff
@@ -28,33 +28,64 @@ import (
...
+}
+
+func NewLocalClient(client pkgclient.Client) Clients {
+	return &LocalClient{
+		client: client,
+	}
+}
+
+// Client returns the local k8s client
+func (c *LocalClient) Client(token string) (pkgclient.Client, error) {
```
[the remark this answers is not in the document]
Comment (WangXiangUSTC) [reply pkg/clientpool/client.go:50]: just used to implement the `Clients` interface

**Review by g1eny0ung — APPROVED** [review]
LGTM

File: helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml [hunk 2]
```diff
@@ -52,6 +52,8 @@ spec:
               value: {{ .Values.controllerManager.targetNamespace }}
             - name: CLUSTER_SCOPED
               value: "{{ .Values.clusterScoped }}"
+            - name: UNSAFE_MODE
```
Comment (zhouqiang-cl) [inline helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml:55]: How about use security_mode

**Comment by zhouqiang-cl** [comment]
And I think we should also need to add an option in `install.sh` and in documents

**Review by fewdan — DISMISSED** [review]
LGTM

File: helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml (hunk 2 above)
Comment (WangXiangUSTC) [reply helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml]: done

**Comment by WangXiangUSTC** [comment]
> And I think we should also need to add an option in `install.sh` and in documents

I update securityMode=false in `install.sh`, PTAL again @zhouqiang-cl

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**